src/stores/MemberListStore.ts

```diff
@@ -70,7 +70,7 @@ export class MemberListStore {
             return [];
         }
 
-        if (!this.isLazyLoadingEnabled(roomId) || this.loadedRooms.has(roomId)) {
+        if (!(await this.isLazyLoadingEnabled(roomId)) || this.loadedRooms.has(roomId)) {
```

Comment:
Can we reverse this if condition so that the synchronous path is not blocked behind the async one?